src/main/java/com/purbon/kafka/topology/actions/topics/SyncTopicAction.java

```diff
@@ -43,20 +46,27 @@ public String getTopic() {
 
   @Override
   public void run() throws IOException {
-    syncTopic(topic, fullTopicName, listOfTopics);
+    syncTopic(topic, fullTopicName);
   }
 
-  private void syncTopic(Topic topic, String fullTopicName, Set<String> listOfTopics)
-      throws IOException {
-    LOGGER.debug(String.format("Sync topic %s", fullTopicName));
-    if (existTopic(fullTopicName, listOfTopics)) {
+  private boolean topicExists() {
+    return topicExists;
+  }
+
+  private void syncTopic(Topic topic, String fullTopicName) throws IOException {
+    LOGGER.debug("Sync topic {}", fullTopicName);
+    if (topicExists()) {
       if (topic.partitionsCount() > adminClient.getPartitionCount(fullTopicName)) {
-        LOGGER.debug(String.format("Update partition count of topic %s", fullTopicName));
+        LOGGER.debug("Update partition count of topic {}", fullTopicName);
         adminClient.updatePartitionCount(topic, fullTopicName);
+      } else {
+        LOGGER.warn(
```

Comment:
well, that depends, it should point out (warn you) that your actual config will not be applied :-), so for me info is something I can read, skip, forget or simply don't react on. if it is warning I know I have to have a closer look.

Reply:
Fair enough, however, the warn here is not necessary, as it is usually used to signal potential problems (when the problem cause is not clear), so please keep it to *info*, please. 
Context: This is mostly informative, you might like to check and change your config, but is certainly not going to break anything for you.

Do you see it otherwise?

Reply:
ok, will change to info.